client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/test/index.js

```diff
@@ -172,16 +173,6 @@ describe( 'Create shipping label button', () => {
 		] );
 	} );
 
-	it( 'should show a busy loading state when installing or activating ', () => {
-		shippingBannerWrapper.setProps( {
-			isRequesting: true,
-		} );
-		const createShippingLabelButton = shippingBannerWrapper.find( Button );
-		expect( createShippingLabelButton.length ).toBe( 1 );
-		expect( createShippingLabelButton.prop( 'disabled' ) ).toBe( true );
-		expect( createShippingLabelButton.prop( 'isBusy' ) ).toBe( true );
-	} );
-
```

Comment:
Testing this in line 264 `describe( 'In the process of installing, activating, loading assets for WooCommerce Service', () => {` test case instead.